packages/zero-client/src/client/context.ts

```diff
@@ -43,12 +46,12 @@ export class ZeroContext implements QueryDelegate {
   constructor(
     lc: LogContext,
     mainSources: IVMSourceBranch,
-    addQuery: AddQuery,
+    queryManager: QueryManager,
```

Comment:
I think maybe `ZeroContext` isn't buying us much and should just be folded into `Zero`. I get why `UpdateManager` exists - it encapsulates some logic. But `ZeroContext` doesn't really seem to play a role that's clearly separate from `Zero`.